src/components/Tables/ReactiveTableRow/ReactiveTableRow.tsx

```diff
@@ -25,11 +25,11 @@ const ReactiveTableRow = ({
   onClick,
 }: ReactiveTableRowProps) => {
   const backgroundColor = index % 2 === 0 ? STYLES.COLORS.EVEN_CELL : STYLES.COLORS.ODD_CELL;
+  const isMobile = Boolean(useMediaQuery({ query: `(max-width: ${screenWidths.tablet})` }));
```

Comment:
The linter is unhappy with it unwrapped.

Reply:
The linter shouldn't complain about this particular call of `useMediaQuery` because it's called at the [top-level](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level) inside a component though?